app/models/goal.py

```diff
@@ -3,3 +3,14 @@
 
 class Goal(db.Model):
```

Comment:
For the models it also might be useful to have a helper class-method to create an instance of the model from a dictionary (like request_body)